app/api/users.py

```diff
@@ -237,6 +238,13 @@ def before_get_object(self, view_kwargs):
             else:
                 view_kwargs['id'] = None
 
+        if view_kwargs.get('group_id') is not None:
+            group = safe_query_kwargs(Group, view_kwargs, 'group_id')
+            if group.user is not None:
+                view_kwargs['id'] = group.user.id
```

Comment:
```suggestion
            if group.user_id is not None:
                view_kwargs['id'] = group.user_id
```

group.user will fetch the user from DB